deploy/zfs-operator.yaml

```diff
@@ -2204,11 +2206,11 @@ spec:
           image: openebs/zfs-driver:ci
           imagePullPolicy: IfNotPresent
           args:
-            - "--nodeid=$(OPENEBS_NODE_ID)"
+            - "--nodeid=$(OPENEBS_NODE_NAME)"
```

Comment:
Is it node name or node ID as the arg?

Reply:
Yes, better to rename the arg here, will do that. Thanks